iot-hub/Quickstarts/device-streams-proxy/device-streams-proxy.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
As per our offline conversation, lets meet to discuss the right home for these scenarios. At the moment because of time pressure, I'm going to let this in. We will reconvene later and decide.